### PR TITLE
test: add missing tests for #182

### PR DESCRIPTION
**Pull request by silkentrance**
chore: update rimraf to latest version
chore: remove duplicate dependencies section from package.json

### CODE REVIEW

**Comment by silkentrance** [comment]
See also #182, #183

File: test/file-sync-test.js [hunk 1]
```diff
@@ -44,6 +44,10 @@ describe('tmp', function () {
     });
 
     describe('when running issue specific inband tests', function () {
+      it('on issue #182: should not replace empty postfix with ".tmp"', function () {
+        const tmpobj = tmp.fileSync({ postfix: '' });
+        assert.ok(tmpobj.name.indexOf('.tmp') === -1);
```
Comment (raszi) [inline test/file-sync-test.js]: What about:

```suggestion
        assert.ok(!tmpobj.name.endsWith('.tmp'));
```

File: test/file-test.js [hunk 2]
```diff
@@ -54,6 +54,16 @@ describe('tmp', function () {
     });
 
     describe('when running issue specific inband tests', function () {
+      it('on issue #182: should not replace empty postfix with ".tmp"', function (done) {
+        tmp.file({ postfix: '' }, function (err, path) {
+          try {
+            assert.ok(path.indexOf('.tmp') === -1);
```
Comment (raszi) [inline test/file-test.js]:
```suggestion
            assert.ok(!path.endsWith('.tmp'));
```

File: package.json [hunk 3]
```diff
@@ -22,9 +22,8 @@
     "node": ">=6"
   },
   "dependencies": {
-    "rimraf": "^2.6.2"
+    "rimraf": "^2.6.3"
   },
-  "dependencies": {},
```
Comment (raszi) [inline package.json]: Nice catch!

File: test/file-sync-test.js (hunk 1 above)
Comment (silkentrance) [reply test/file-sync-test.js]: done, good find

File: test/file-test.js (hunk 2 above)
Comment (silkentrance) [reply test/file-test.js]: done, good find

**Comment by silkentrance** [comment]
Merging now, as all change requests have been met.

**Pull request merged**